config.sh

```diff
@@ -14,7 +14,7 @@ CF_CLI_RELEASE_VERSION7=7.7.1
 CREDHUB_CLI_RELEASE_VERSION=2.9.18
 # bosh-cli gets really slow at 6.2.0, so we're pinned here.
 # check https://github.com/cloudfoundry/bosh-cli/issues/570 for details
-BOSH_CLI_V2_RELEASE_VERSION=7.2.3
+BOSH_CLI_V2_RELEASE_VERSION=7.5.4
```

Comment:
Since we're not pinned to < 6.2.0 anymore, can we remove the comments above this line?

Reply:
Thanks Mark, I had initially deleted those but I missed actually pushing the commit. Those comments are now removed